pyscripts/horton_create_identities.py

```diff
@@ -11,149 +11,95 @@
 import json
 import pathlib
 import traceback
+import argparse
 from colorama import init, Fore, Back, Style
 from os.path import dirname, join, abspath
 sys.path.insert(0, abspath(join(dirname(__file__), '../horton_helpers')))
```

Comment:
should not be necessary if you've called setup-python.sh (or if you've called pip install -e ../horton_helpers)

Reply:
Fixed.